src/app/admin/wallet-activation/project-activation/project-activation.component.html

```diff
@@ -1,10 +1,10 @@
 <div *ngIf="projects !== undefined">
-    <h5 class="explainer-mini">
+    <h5>
         {{ 'admin.activations.projects.title' | translate }}
     </h5>
     <div *ngIf="projects.length > 0" class="mt-3">
         <ul class="list-group">
-            <li *ngFor="let project of projects" class="list-group-item explainer-mini link-remove-style">
```

Comment:
There is no reason to add `link-remove-style` class on the list item if you apply that on the anchor link. (that's why I made `nostyle` on a specific link.